docs/source/examples/combine_ddd_zne.md

```diff
@@ -0,0 +1,241 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Composing techniques: Digital Dynamical Decoupling and Zero Noise Extrapolation
+
+Noise in quantum computers can arise from a variety of sources, and sometimes applying multiple error mitigation techniques can be more beneficial than applying a single technique alone.
+
+Here we apply a combination of Digital Dynamical Decoupling (DDD) and Zero Noise Extrapolation (ZNE) on a GHZ state.
```

Comment:
It would be nice to give some intuition on why GHZ state preparation has been picked for this example. It becomes more clear later in the notebook, but it'd be nice for the reader to have something upfront.

Reply:
Done